README.md

```diff
@@ -43,13 +43,29 @@ pip3 install pyro-ppl
 ```sh
 git clone git@github.com:uber/pyro.git
 cd pyro
+git checkout master  # master is pinned to the latest release
 pip install .
 ```
 
-Note that to use Pyro's `dev` branch, you will need to build PyTorch's master branch. 
-Refer to the [README](https://github.com/pytorch/pytorch/blob/master/README.md) for more details
-on how to build PyTorch from source. Otherwise, to use the latest version of Pyro that 
-is compatible with PyTorch release, use the `master` branch of Pyro.
+## Installing Pyro dev branch
+
+For recent features you can install Pyro from source.
+
+First install a recent PyTorch, currently PyTorch commit `ca5071d07`.
```

Comment:
the most recent commit will change, do you want to just put a placeholder `[commit hash]`?

Reply:
I think it would be better if we specify the exact commit that was used to build the PyTorch CPU wheels used in travis, so as to have a certain level of assurance that the integration isn't broken. Every time we update the wheels, we can update this README.

Reply:
> the exact commit that was used to build the PyTorch CPU wheels

Yes that is my intention. The commit in this PR was the latest wheel commit.